boards/xtensa/intel_s1000_crb/Kconfig.defconfig

```diff
@@ -167,6 +145,38 @@ config GPIO_DW_0
 
 endif
 
+if FLASH
+
+config SPI
+	def_bool y
+
+config SPI_NOR
+	def_bool y
+
+config SPI_NOR_JEDEC_ID
+	default 0xC2803A
+
+config SPI_NOR_PAGE_SIZE
+	default 256
+
+config SPI_NOR_SECTOR_SIZE
+	default 4096
+
+config SPI_NOR_SECTORS
+	default 1024
+
+choice SPI_NOR_BLOCK_ERASE_SIZE
+	default SPI_NOR_BLOCK_ERASE_64K
+endchoice
+
+config FLASH_HAS_PAGE_LAYOUT
```

Comment:
What is the concept behind "page layout"?

Reply:
This is nothing but the support for different partitions (boot, slot0, slot1 and scratch area) required by mcubootloader.